src/routers/alpha-router/alpha-router.ts

```diff
@@ -119,51 +119,91 @@ export class AlphaRouter implements IRouter<AlphaRouterConfig>, ISwapToRatio<Alp
     token0Balance: CurrencyAmount,
     token1Balance: CurrencyAmount,
     position: Position,
+    errorTolerance: Fraction,
     swapConfig?: SwapConfig,
     routingConfig = DEFAULT_CONFIG
   ): Promise<SwapRoute<TradeType.EXACT_INPUT> | null> {
-      if (
-        token0Balance.currency.wrapped.address.toLowerCase()
-        > token1Balance.currency.wrapped.address.toLowerCase()
-      ) {
+      const maxIterations = 5
```

Comment:
Can we make this parameterized? Will be good if we can play with this in the interface/routing-api without needing to do a release of the smart-order-router package.

Maybe a new SwapAndAddConfig object?